lib/aligning_guidelines.js

```diff
@@ -84,8 +85,8 @@ function initAligningGuidelines(canvas) {
       var objectCenter = canvasObjects[i].getCenterPoint(),
           objectLeft = objectCenter.x,
           objectTop = objectCenter.y,
-          objectHeight = canvasObjects[i].getBoundingRectHeight() / viewportTransform[3],
-          objectWidth = canvasObjects[i].getBoundingRectWidth() / viewportTransform[0];
+          objectHeight = canvasObjects[i].getBoundingRect().height / viewportTransform[3],
+          objectWidth = canvasObjects[i].getBoundingRect().width / viewportTransform[0];
```

Comment:
please use the same logic here, call boundingRect once, and then read width/height.